packages/server/.env.example

```diff
@@ -71,3 +71,8 @@ PORT=3000
 # GLOBAL_AGENT_HTTP_PROXY=CorporateHttpProxyUrl
 # GLOBAL_AGENT_HTTPS_PROXY=CorporateHttpsProxyUrl
 # GLOBAL_AGENT_NO_PROXY=ExceptionHostsToBypassProxyIfNeeded
+##AWS Secrets Manager
+# AWS_ACCESS_KEY_ID=<your-access-key>
+# AWS_SECRET_ACCESS_KEY=<your-secret-key>
+# USE_AWS_SECRETS_MANAGER=true
```

Comment:
I'd prefer to have more generic name that can be reused in future for Azure Key Vault, Okta and other else.

Perhaps: `SECRET_MANAGER_PROVIDER`, the value can be `aws`, `azure`, etc

Also, search for the variable `STORAGE_TYPE`, you can similar references and other places that we need to declare these env variables as well